src/Glpi/Controller/ErrorController.php

```diff
@@ -160,6 +162,14 @@ private function getErrorResponse(\Throwable $exception, Request $request): Resp
             }
         }
 
+        switch (true) {
+            case $exception instanceof MaintenanceException:
+                return new Response(TemplateRenderer::getInstance()->render('maintenance.html.twig', [
```

Comment:
We should take advantage of this PR to send a 503 response code.

Reply:
Hmm, I'm not sure. To me, HTTP 503 is more related to an app stack being unavailable. Here, the stack is available but the database is in update process. If PHP wasn't working at all but the HTTP server was fine, a 503 would be okay (though an HTTP 504 would be also relevant).

Do we really need crawlers or monitoring tools to log maintenance as an error?

Reply:
I will ask our cloud team sysadmins what is the best option for them. Having a 200 error code could be a problem too, as monitoring tools will not be able to detect when the maintenance mode has not been disabled after the maintenance time frame.

Reply:
Will wait for ops feedback then 👌 In the meantime, I'll change for 503, there might be more reasons for it than for 200

Reply:
I just confirmed with @CupidSG . Having a 503 error code would be better tha,n the current 200 OK code.